spec/features/shelters/shelters_spec.rb

```diff
@@ -5,11 +5,11 @@
   it 'the shelters index page shows a list of all the shelters' do 
     
       info1 = {:name => 'Rocky Mountain Puppy Rescue', :address => '11 Market Street' , :city => 'Denver' , :state => 'CO', :zip => '80203'} 
-      shelter1 = Shelter.create(info1)
+      shelter1 = Shelter.create!(info1)
```

Comment:
add the bang operator to error out when creation is prevented by the number of arguments passed in